x-pack/plugins/ingest_pipelines/public/application/components/pipeline_form/pipeline_form.tsx

```diff
@@ -74,6 +78,11 @@ export const PipelineForm: React.FunctionComponent<PipelineFormProps> = ({
     if (processorStateRef.current) {
       const state = processorStateRef.current;
       if (await state.validate()) {
+        // We only want to show unsaved changed prompts to the user after the form
```

Comment:
Is this comment correct - do you mean we **do not** want to show the unsaved changes prompts if the form has been submitted?